_episodes/02-cupy.md

```diff
@@ -71,6 +71,11 @@ The result of this should be four times the content of `primary_unit`.
 
 The computation we want to perform on this image is a convolution, both on the host and device and compare the results and execution times.
 **TODO** explain in simple language what a convolution is and what it does.
+
+An example of a convolution can be seen on the figure below (illustration by Michael Plotke, CC BY-SA 3.0, via Wikimedia Commons).
```

Comment:
I would write

`An example of convolution can be seen in the figure below`

Reply:
Corrected in the last commit.